src/components/InputMediaGroup/InputMedia/InputMedia.types.ts

```diff
@@ -1,13 +1,13 @@
-export interface IMedia {
+export type IMedia = {
   file: File;
   id: string;
-}
+};
 
-export interface IInputMediaProps {
+export type IInputMediaProps = {
```

Comment:
Tá parecendo algo que o lin/prettier mudou, ai é necessário mudar o prefixo: 
```suggestion
export type TInputMediaProps = {
```